### PR TITLE
Translations, contact info and copyright to main page

### PR DESCRIPTION
**Pull request by lurppa**

### CODE REVIEW

File: i18n/fi.yaml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  translation: "Sähköposti"
+
+- id: contactMessage
+  translation: "Viesti"
+
+- id: contactSend
+  translation: "Lähetä"
+
+- id: navHome
+  translation: "siirry kotisivulle"
```
Comment (45spoons) [inline i18n/fi.yaml:35]: Letter casing here inconsistent

File: i18n/fi.yaml [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+  translation: "Tietoja"
+
+- id: newer
+  translation: "Uudempi"
+
+- id: older
+  translation: "Vanhempi"
+
+- id: publishedOn
+  translation: ""
```
[the remark this answers is not in the document]
Comment (lurppa) [reply i18n/fi.yaml:71]: Here's how it is defined in `themes/hugo-universal-theme/i18n/en.yaml`:
```yaml
- id: publishedOn
  translation: "on"
```
and how it is used in `themes/hugo-universal-theme/layouts/partials/recent_posts.html`:
```html
{{ $createdAt := .Date.Format .Site.Params.date_format }}
{{ range $index, $month := slice "January" "February" "March" "April" "May" "June" "July" "August" "September" "October" "November" "December" }}
{{ $createdAt = replace $createdAt $month (i18n $month) }}
{{ end }}
{{ i18n "publishedOn" }} {{ $createdAt }}
```
I don't think that in the context of where this translation shows up it should display anything in Finnish.

File: i18n/fi.yaml (hunk 1 above)
Comment (lurppa) [reply i18n/fi.yaml:35]: Same thing with the translations context in here. How it's defined in `themes/hugo-universal-theme/i18n/en.yaml`:
```yaml
- id: navHome
  translation: "go to homepage"
```
and how the translation is used in `themes/hugo-universal-theme/layouts/partials/nav.html`:
```html
<span class="sr-only">{{ .Title }} - {{ i18n "navHome" }}</span>
```

File: i18n/fi.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (45spoons) [reply i18n/fi.yaml:71]: Ahaa yes that is clear now! 👍

File: i18n/fi.yaml (hunk 1 above)
Comment (45spoons) [reply i18n/fi.yaml:35]: Alright I see, thanks for taking the time to point that out! 😊

**Review by 45spoons — APPROVED** [review]

File: i18n/fi.yaml (hunk 1 above)
Comment (lurppa) [reply i18n/fi.yaml:35]: It's all good 🙂

**Pull request merged**